index.html

```diff
@@ -0,0 +1,451 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <nav>
+        <div class="menu-bars">
+          <i class="fas fa-bars"></i>
+        </div>
+        <div class="closed-menu-bars">
+          <i class="fa fa-close"></i>
+        </div>
+        <ul class="desktop-nav d-flex">
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
```

Comment:
Great work so far in your design, it looks almost like the Figma design! but, to make it very similar I think, it would be better to remove the envelope icon on your contact link.

- [ ] your menu : 
![image](https://user-images.githubusercontent.com/71874425/152647515-73a2622f-a67b-4cb3-acc6-770628cb03a6.png)
- [ ] Correct design from figma: 
![image](https://user-images.githubusercontent.com/71874425/152647546-397c2133-e687-4b9b-8515-cb7ef4b0b652.png)

Reply:
- Yes, you are right.
- Thanks for contributing to my professionalism